packages/server/graphql/mutations/helpers/notifications/makeMattermostAttachments.ts

```diff
@@ -0,0 +1,58 @@
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import appOrigin from '../../../../appOrigin'
+import {PALETTE} from '~/styles/paletteV3'
+
+export interface ButtonSpec {
+  label: string
+  link: string
+}
+
+export interface Field {
+  short: boolean
+  title?: string
+  value: string
+}
+
+export interface FieldAttachmentHeader {
+  fallback: string
+  color?: string
+  title?: string
+  title_link?: string
+  author_name?: string
+  author_link?: string
+  author_icon?: string
+}
+
+// type FieldAttachment = FieldAttachmentHeader & {fields: Array<Field>}
+
+interface FieldAttachment extends FieldAttachmentHeader {
+  fields: Array<Field>
+}
+
+export const makeFieldsAttachment = (
+  fields: Array<Field>,
+  header?: FieldAttachmentHeader
+): FieldAttachment => {
+  const defaultHeader = {
+    fallback: 'Parabol has a message for you',
+    author_icon: makeAppURL(appOrigin, `static/images/brand/mark-color%403x.png`),
```

Comment:
this will grab the icon from our webserver instead of our CDN.
it'd be worthwhile making it grab the icon from our CDN to reduce traffic to our poor little webserver, especially because mattermost doesn't close the icon. they just fetch it from us every time it loads.

to do that, at the top of this file, simply import the image:
```
import authorIcon from '../../../../../../static/images/brand/mark-color@3x.png'
...
author_icon: authorIcon
```
webpack takes care of the rest 🎉  🪄

Reply:
still a bug!